### PR TITLE
Add store results to circleci config / Testing branch protection with CircleCI

### PR DESCRIPTION
**Pull request by ryancmoniz**
Finish setting up CircleCI configuration with master branch protection enabled and to only allow merges on status check passes.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -15,4 +15,6 @@ jobs:
           name: Running server in background
           command: npm run start
           background: true
-      - run: npm run test
\ No newline at end of file
+      - run: npm run test
+      - store_test_results:
+          path: test-results
```
Comment (ingridclaire) [inline .circleci/config.yml:20]: is this a file called test-results that you add to store the results?

File: README.md [hunk 2]
```diff
@@ -1,44 +1,3 @@
...
-
-* **Joshua Kim** - *Initial work* - [joshskkim](https://github.com/joshskkim)
-
-* *See also the rest of the modules done by our team!*
-* **FEC-Team Gandalf** (https://github.com/Team-Gandalf)
-
-
-* **Haider Syed** - *Sidebar and Body component* (https://github.com/Team-Gandalf/sidebar-body-related) - [sfshaider](https://github.com/sfshaider)
-* **Brian Hampton** - *Media component* (https://github.com/Team-Gandalf/Media) - [BHamp](https://github.com/BHamp)
-* **Sajjan Raj Vaidya** - *Announcements component* (https://github.com/Team-Gandalf/announcements) - [sajjanrajvaidya](https://github.com/sajjanrajvaidya)
```
Comment (ingridclaire) [inline README.md:44]: So you deleted the contents of the original readme? mine didn't have one so just wondering..

Comment (ryancmoniz) [reply README.md:44]: Yes, the original readme had a lot of info.

File: .circleci/config.yml (hunk 1 above)
Comment (ryancmoniz) [reply .circleci/config.yml:20]: Yes, it creates a file on your CircleCI server that you can view previous test results with.

**Pull request merged**